### PR TITLE
Remove unnecessary elem/block in normalize v2

### PR DESCRIPTION
**Pull request by Vittly**
#332

### CODE REVIEW

File: packages/decl/test/formats/v2/normalize/unusual.test.js [hunk 1]
```diff
@@ -56,7 +56,6 @@ describe('normalize2.unusual', () => {
 
         expect(normalize(decl).map(simplifyCell)).to.deep.equal([
             { entity: { block: 'block' }, tech: null },
-            { entity: { block: 'block', elem: 'elem1' }, tech: null },
```
Comment (Vittly) [inline packages/decl/test/formats/v2/normalize/unusual.test.js]: It is too complex for me. Is it okey?

Comment (qfox) [reply packages/decl/test/formats/v2/normalize/unusual.test.js]: Yeah.

Read this like:
```
block: 'block',
elem: 'elem1',
mod: 'mod1',
val: 'v1',
```
and after:
```
             elems: {
                 elem: 'elem2'
             },
             mods: {
                 mod2: 'v2'
             }
```

Semantically it's the same as:
```
{
  block: 'block', elem: 'elem1', mod: 'mod1', val: 'v1',

  shouldDeps: [
    { block: 'block', elem: 'elem2' },
    { block: 'block', elem: 'elem1', mod: 'mod2', val: 'v2' },
  ]
}
```
→→→
```
block__elem1_mod1_v1
→ block__elem2
→ block__elem1_mod2_v2
```

`{ entity: { block: 'block' }, tech: null },` is redundant too.

Comment (qfox) [reply packages/decl/test/formats/v2/normalize/unusual.test.js]: Guess, there is a problem with recursive `processMods({ block, mods: mod, tech });` that adding pure block entity. Probly I'm wrong.

File: packages/decl/test/formats/v2/normalize/mod-mods-vals.test.js [hunk 2]
```diff
@@ -41,7 +40,6 @@ describe('normalize2.mod-mods-vals', () => {
         const decl = { mods: { mod: 'val' } };
 
         expect(normalize(decl, scope).map(simplifyCell)).to.deep.equal([
-            { entity: { block: 'sb' }, tech: null },
```
Comment (Vittly) [inline packages/decl/test/formats/v2/normalize/mod-mods-vals.test.js]: Is it wrong? Isn't `normalize({ block: 'b', mods: { m: 'v' })` same as `normalize({ mods: { m: 'v' }), scope({ block: 'b' }))` ?

File: packages/decl/test/formats/v2/normalize/elems.test.js [hunk 3]
```diff
@@ -82,7 +82,6 @@ describe('normalize2.elems', () => {
         };
 
         expect(normalize(decl, { entity: { block: 'sb' } }).map(simplifyCell)).to.deep.equal([
-            { entity: { block: 'sb' }, tech: null },
             { entity: { block: 'sb', elem: 'close' }, tech: null },
```
Comment (Vittly) [inline packages/decl/test/formats/v2/normalize/elems.test.js]: IMHO it is not okey

File: packages/decl/test/formats/v2/normalize/elems-mods.test.js [hunk 4]
```diff
@@ -87,7 +87,6 @@ describe('normalize2.elems-mods', () => {
         ];
 
         expect(normalize(decl, { entity: { block: 'sb' } }).map(simplifyCell)).to.deep.equal([
-            { entity: { block: 'sb' }, tech: null },
```
Comment (Vittly) [inline packages/decl/test/formats/v2/normalize/elems-mods.test.js]: is it ok?

File: packages/decl/test/formats/v2/normalize/mod-mods-vals.test.js (hunk 2 above)
Comment (qfox) [reply packages/decl/test/formats/v2/normalize/mod-mods-vals.test.js]: yep, it's wrong

File: packages/decl/test/formats/v2/normalize/mod-mods-vals.test.js [hunk 5]
```diff
@@ -20,7 +20,6 @@ describe('normalize2.mod-mods-vals', () => {
         expect(normalize(decl, scope).map(simplifyCell)).to.deep.equal([
             { entity: { block: 'sb', modName: 'mod', modVal: true }, tech: null },
             { entity: { block: 'sb', modName: 'mod', modVal: 'val' }, tech: null },
-            { entity: { block: 'sb' }, tech: null },
```
Comment (qfox) [inline packages/decl/test/formats/v2/normalize/mod-mods-vals.test.js]: was better

File: packages/decl/test/formats/v2/normalize/elems.test.js [hunk 6]
```diff
@@ -82,7 +82,6 @@ describe('normalize2.elems', () => {
         };
 
         expect(normalize(decl, { entity: { block: 'sb' } }).map(simplifyCell)).to.deep.equal([
-            { entity: { block: 'sb' }, tech: null },
```
Comment (qfox) [inline packages/decl/test/formats/v2/normalize/elems.test.js]: wrong

File: packages/decl/test/formats/v2/normalize/elems-mods.test.js (hunk 4 above)
Comment (qfox) [reply packages/decl/test/formats/v2/normalize/elems-mods.test.js]: nope

File: packages/deps/test/formats/deps.js/parser.test.js [hunk 7]
```diff
@@ -83,7 +83,6 @@ describe('parser', () => {
                 shouldDeps: [{ tech: 'bemhtml', elems: ['e1', 'e2'] }, { tech: 'i18n', block: 'translations' }]
             }]
         }])).to.deep.equal([
-            'be.tmpl-spec.js -> be.bemhtml',
```
Comment (qfox) [inline packages/deps/test/formats/deps.js/parser.test.js]: was correct

**Comment by Vittly** [comment]
fixed

**Comment by qfox** [comment]
Вольем?

**Review by qfox — APPROVED** [review]

**Pull request merged**